01-shapefile-attributes.Rmd

```diff
@@ -1,15 +1,18 @@
 ---
 layout: post
-title: "Lesson 01: Work With Shapefile Attributes in R"
+title: "Lesson 01: Shapefile Metadata & Attributes in R"
```

Comment:
Based on 1 undergrad class and then my own playing around with data I know: shapefile was a spatial data file and what vectors were.  However, I didn't realize that shapefiles were only for vectors and not for raster files. (Not sure if that helps, nor if I'm normal for people approaching this data).